app/controllers/search_controller.rb

```diff
@@ -0,0 +1,7 @@
+class SearchController < ApplicationController
+  def index
```

Comment:
See my comments in your Facade. If you were to implement that suggestion, your controller could look like:
```
def index
  @query = params[:query]
  @result_count, @top_results = QuotableFacade.top_results(@query)
end
```
Now your view has access to the word, the total count, and quote results for your view.

Reply:
Ohhh, How exactly does that `@result_count, @top_results = QuotableFacade.top_results(@query)` line work?